### ISSUE TITLE
Could we include photos of the polling stations? 

### ISSUE THREAD

**Issue by JoeMitchell** [DemocracyClub/UK-Polling-Stations#415]
A la Google Streetview previews on Rightmove or Citymapper

![image](https://cloud.githubusercontent.com/assets/8096305/20896279/e0fc23d8-bb14-11e6-8f47-ae1130e72d9c.png)

Or, could we at least put some photos up of the Polling Station signage and say 'look out for these type of signs attached to buildings or fences near the polling station'?

**Comment by chris48s** [comment]
<img src="http://www.rochdale.gov.uk/rbcimages/2015-04-24-how-to-vote.jpg"></img>

**Comment by JoeMitchell** [comment]
Yup! That would do for a start! Is that on the back of a poll card or something?

**Comment by chris48s** [comment]
No it is from Rochdale Council's website ( www.rochdale.gov.uk/council-and-democracy/elections/Pages/polling-stations.aspx ), but it has quite a pleasing Ikea assembly instructions quality to it, doesn't it :)

Probably can't steal that, but I reckon something 'generalised' like that is orders of magnitude more achievable than streetview.

Streetview probably isn't do-able in an automated way - In most cases we **do** know the right co-ordinates to drop the camera at (we'll abstract that sometimes we use a postcode centroid and that sometimes the point we've got isn't necessarily just outside the front door or whatever), but streetview is a 360 view and we have no way to know which direction to point. With no manual intervention (just dropping the camera at a point facing in an unspecified direction) there's a greater chance that the thing we're trying to point at is out of shot rather than in front of the camera.. which isn't that useful

Doing something like that would require a manual intervention for every station. That works if you're an estate agent and you have to add in the details if every property you're selling anyway, but not if you need to bulk-import a bunch of data about geographic locations you don't know and haven't been to..

**Comment by symroe** [comment]
@memespring has thought about this sort of thing in the past. We might not need to actually point at the polling station – it might be enough to just link to the street view of the area, to allow people to explore it – especially if they are new to the area.

Also to consider: temporary polling stations (in a portacabin or whatever) that might not be in street view.
